core/src/main/java/org/kestra/core/tasks/scripts/Bash.java

```diff
@@ -99,38 +112,101 @@
         },
         dynamic = true
     )
-    private List<String> files;
+    protected List<String> outputs;
+
+    @InputProperty(
+        description = "Input files are extra files supplied by user that make it simpler organize code.",
+        body = {
+            "Describe a files map that will be written and usable in execution context. In python execution context is in a temp folder, for bash scripts, you can reach files using a scriptFolder variable like 'source {{scriptFolder}}/myfile.sh' "
```

Comment:
Maybe we should add some variables instead of this ? 
Like `{{ inputsFiles.name }}` to support bash properly. 

Also remove the comment for python, and add it to python Task please